pkg/deployments/tasks/20220704-timelock-authorizer/input/mainnet.ts

```diff
@@ -0,0 +1,250 @@
+import { DAY } from '@balancer-labs/v2-helpers/src/time';
+import Task, { TaskMode } from '../../../src/task';
+import { flatten } from 'lodash';
+import { DelayData, RoleData } from './types';
+import { ANY_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+
+const EVERYWHERE = ANY_ADDRESS;
+
+const Vault = new Task('20210418-vault', TaskMode.READ_ONLY, 'mainnet');
+const ProtocolFeesCollector = new Task('20210418-vault', TaskMode.READ_ONLY, 'mainnet');
+const ProtocolFeesWithdrawer = new Task('20220517-protocol-fee-withdrawer', TaskMode.READ_ONLY, 'mainnet');
+
+const BalancerTokenAdmin = new Task('20220325-balancer-token-admin', TaskMode.READ_ONLY, 'mainnet');
+const BalancerMinter = new Task('20220325-gauge-controller', TaskMode.READ_ONLY, 'mainnet');
+const GaugeAdder = new Task('20220628-gauge-adder-v2', TaskMode.READ_ONLY, 'mainnet');
+const GaugeController = new Task('20220325-gauge-controller', TaskMode.READ_ONLY, 'mainnet');
+const VotingEscrowDelegationProxy = new Task('20220325-ve-delegation', TaskMode.READ_ONLY, 'mainnet');
+const SmartWalletChecker = new Task('20220420-smart-wallet-checker', TaskMode.READ_ONLY, 'mainnet');
+const LiquidityGaugeV5 = new Task('20220325-mainnet-gauge-factory', TaskMode.READ_ONLY, 'mainnet');
+
+const BalancerRelayer = new Task('20211203-batch-relayer', TaskMode.READ_ONLY, 'mainnet');
+const LidoRelayer = new Task('20210812-lido-relayer', TaskMode.READ_ONLY, 'mainnet');
```

Comment:
Is this still used?

Reply:
Yes, the frontend still uses this for wrapping stETH.